src/App.js

```diff
@@ -0,0 +1,32 @@
+import './App.css';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
+import Header from './components/Header';
+import Rockets from './components/Rockets';
+import Missions from './components/Missions';
+import Profile from './components/Profile';
+
+function App() {
+  return (
+    <Provider store={store}>
+      <Router>
+        <Header />
+        <Switch>
+          <Route path="/rockets">
+            <Rockets />
+          </Route>
+          <Route path="/missions">
+            <Missions />
+          </Route>
+          <Route path="/profile">
+            <Profile />
+          </Route>
+        </Switch>
+      </Router>
+    </Provider>
```

Comment:
- [ ] The App is working fine, but the landing page is not rendering anything to prevent that you can add a `useEffect & dispatch` to render the `rockets` on the home page.

![CPT2110071709-558x239](https://user-images.githubusercontent.com/61562626/136424621-9ced8657-4f91-4f26-a746-fdaec07bb37f.gif)

Reply:
Dear @dasleker,

Thanks for the review 👍 

We will fix the issues you mentioned in the comment.